motoko/token_transfer_from/README.md

```diff
@@ -165,18 +149,16 @@ URLs:
     icrc1_ledger_canister: http://127.0.0.1:4943/?canisterId=bnz7o-iuaaa-aaaaa-qaaaa-cai&id=mxzaz-hqaaa-aaaar-qaada-cai
 ```
 
-### Step 6: Verify that the ledger canister is healthy and working as expected by using the command:
-
-:::info 
-
-[Learn more about how to interact with the ICRC-1 ledger](https://internetcomputer.org/docs/current/developer-docs/defi/icrc-1/using-icrc1-ledger#icrc-1-and-icrc-1-extension-endpoints).
+## Step 7: Verify that the ledger canister is healthy and working as expected by using the command
 
-:::
+> ![TIP]
```

Comment:
There is another occurrence here that should be `[!TIP]`

Reply:
This has been fixed already, please refresh the PR